tests/helm/utils.go

```diff
@@ -0,0 +1,173 @@
+package helm
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/go-commons/collections"
+	"github.com/gruntwork-io/go-commons/errors"
+	"github.com/gruntwork-io/go-commons/files"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+// get the chart version from Chart.yaml
+func GetChartVersion() (string, error) {
+	output, err := os.ReadFile(path.Join(chartDirectory, "Chart.yaml"))
+	if err != nil {
+		return "", err
+	}
+	var chartInfo struct {
+		Version string `yaml:"version"`
+	}
+	err = yaml.Unmarshal(output, &chartInfo)
+	if err != nil {
+		return "", err
+	}
+	return chartInfo.Version, nil
+}
+
+// The functions below are copied from terratest for the sole reason of being able to skip the helm dependency update
```

Comment:
Here you write you want to be able to skip helm dependency update, but above in the `setupDependencies` function you call the `helm dependency update` command. I'm probably not understanding something, can you explain?

Reply:
This makes sense, thanks for the explanation. Yeah, the tests should totally support a "skip dependency update" flag to speed up local testing, but this can be a separate PR :+1: